### PR TITLE
Metadata address encoding/decoding command tree issue 

### PR DESCRIPTION
**Pull request by channa-figure**
## Description

The command tree for metadata address parsing was incorrectly setup.  This was causing it to be listed twice in the help command.  This could also have caused other issues down the road.

closes: #231

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [x] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/provenance-io/provenance/blob/main/CONTRIBUTING.md#pr-targeting))
- [x] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [x] Wrote unit and integration [tests](https://github.com/provenance-io/provenance/blob/main/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [x] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [x] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Review `Codecov Report` in the comment section below once CI passes

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -50,7 +50,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 * Update Cosmos SDK to 0.42.3
 * Remove deprecated ModuleCdc amino encoding from name module #189
 * Remove deprecated ModuleCdc amino encoding from attribute module #188
-
+* Fix cli metadata address encoding/decoding command tree #231
```
Comment (iramiller) [inline CHANGELOG.md:53]: Needs to move up into unreleased.

File: cmd/provenanced/cmd/addresses.go [hunk 2]
```diff
@@ -3,16 +3,35 @@ package cmd
...
 	"github.com/spf13/cobra"
 
 	"github.com/provenance-io/provenance/x/metadata/types"
 )
 
+// GetQueryCmd is the top-level command for name CLI queries.
+func AddMetaAddressCmd() *cobra.Command {
+	queryCmd := &cobra.Command{
+		Use:                        "metaaddress",
+		Short:                      "Decode/Encode Metaaddresses commands",
```
Comment (dwedul-figure) [inline cmd/provenanced/cmd/addresses.go:17]: `Aliases: []string{"ma"}` might be nice here. `"mda"` might also be a good one.

Comment (channa-figure) [reply cmd/provenanced/cmd/addresses.go:18]: Just added the `ma` command.  Added `e` and `d` to encoder and decoder.

**Review by dwedul-figure — APPROVED** [review]
LGTM

**Review by dwedul-figure — APPROVED** [review]
LGTM

**Pull request merged**